### PR TITLE
Kses - Part 21

### PR DESCRIPTION
**Pull request by hoseinrafiei**
Related to the https://github.com/eventespresso/event-espresso-core/issues/3976 issue.

Checklist: https://trello.com/c/201HJZZe/1168-ee4-core-testing-checklist-pr-4012

### CODE REVIEW

File: core/helpers/EEH_Form_Fields.helper.php [hunk 1]
```diff
@@ -343,11 +343,9 @@ public static function get_form_fields_array($fields)
      */
     private static function adminHidden($class, $id, $name, $value)
     {
-        $id    = esc_attr($id);
-        $name  = esc_attr($name);
-        $class = esc_attr($class);
         return "
-        <input name='{$name}' type='hidden' id='{$id}' class='{$class}' value='{$value}' />";
+        <input name='" . esc_attr($name) . "' type='hidden' id='" . esc_attr($id) . "' class='" . esc_attr($class) . "' 
+        value='" . esc_attr($value) . "' />";
```
Comment (tn3rb) [inline core/helpers/EEH_Form_Fields.helper.php:348]: these are getting returned, not echoed, so this is fine as long as it doesn't interfere with any escaping that happens downstream...

File: core/helpers/EEH_Tabbed_Content.helper.php [hunk 2]
```diff
@@ -94,10 +96,9 @@ public static function display_admin_nav_tabs($nav_tabs = [])
         foreach ($nav_tabs as $slug => $tab) {
             $tab_content .= self::tab($slug, false, $tab['link_text'], $tab['url'], $tab['css_class']);
         }
-        $aria_label = esc_attr__('Secondary menu', 'event_espresso');
         return "
-        <nav class='nav-tab-wrapper wp-clearfix' aria-label='{$aria_label}'>
-            {$tab_content}
+        <nav class='nav-tab-wrapper wp-clearfix' aria-label='" . esc_attr__('Secondary menu', 'event_espresso') . "'>
+            " . wp_kses($tab_content, AllowedTags::getWithFormTags()) . "
         </nav>
```
Comment (tn3rb) [inline core/helpers/EEH_Tabbed_Content.helper.php:102]: 👍🏻

**Review by tn3rb — APPROVED** [review]
### good stuff 🍉

**Comment by hoseinrafiei** [comment]
@Saam01 

Please test all backend and frontend menus of EE to see if you can find any issue. It's a escaping PR like the ones that you did before using Trello checklist: https://trello.com/c/201HJZZe/1168-ee4-core-testing-checklist-pr-4012

**Comment by Saam01** [comment]
This ticket is taxable option is missing under Event Editor > Available tickets:

image

**Comment by hoseinrafiei** [comment]
Screen Shot 2022-08-31 at 1 55 21 PM

@Saam01 It's working for me. Could you please make sure that you have correct configuration for taxes? Is everything is configured correctly then let me know how to replicate it.

**Comment by Saam01** [comment]
Seems to be working fine after database reset.

I've finished the kses-21 testing.

**Pull request merged**